### PR TITLE
 Remove _typing_limit subroutine because it's never used 

### PR DESCRIPTION
**Pull request by Wabri**
https://progress.opensuse.org/issues/174271

### CODE REVIEW

File: consoles/vnc_base.pm [hunk 1]
```diff
@@ -66,7 +66,9 @@ sub current_screen ($self) {
     return $self->{vnc}->_framebuffer;
 }
 
-sub _typing_limit () { $bmwqemu::vars{VNC_TYPING_LIMIT} // VNC_TYPING_LIMIT_DEFAULT || 1 }
+sub _typing_limit () {
```
Comment (perlpunk) [inline consoles/vnc_base.pm:69]: It seems this subroutine is never used anywhere. Maybe we should remove it instead.

Comment (Wabri) [reply consoles/vnc_base.pm:69]: Also the #2590 subroutine was never used, but we decided to keep that. I've not strong opinion whether keep it or not.

Comment (perlpunk) [reply consoles/vnc_base.pm:69]: `_typing_limit` has a leading underscore, so by convention it is a subroutine that should only be called from that package. So if we don't use it in that package, IMHO we should remove it. Or alternatively use it in the places where we currently use `VNC_TYPING_LIMIT_DEFAULT`.

Comment (perlpunk) [reply consoles/vnc_base.pm:69]: `mutex_wait` is actually used: https://github.com/search?q=repo%3Aos-autoinst%2Fos-autoinst-distri-opensuse%20mutex_wait&type=code

Comment (Martchus) [reply consoles/vnc_base.pm:69]: Yes, just because it isn't used within os-autoinst itself it doesn't mean it isn't used.

Comment (okurz) [reply consoles/vnc_base.pm:69]: The removal of `_typing_limit` was overlooked in 007c1837. Please remove the function

**Review by okurz — CHANGES_REQUESTED** [review]

File: consoles/vnc_base.pm (hunk 1 above)
Comment (Wabri) [reply consoles/vnc_base.pm]: Ok, done it!

**Review by perlpunk — APPROVED** [review]

**Comment by perlpunk** [comment]
The PR title still needs to be updated

**Review by kalikiana — APPROVED** [review]

**Comment by Wabri** [comment]
I've fixed the typo on the commit and updated the title!

**Review by okurz — APPROVED** [review]

**Pull request merged**